### PR TITLE
CIV-12886 settlement court decides in favour of defendant

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/CIV-12886

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: dashboard-notifications/src/main/java/uk/gov/hmcts/reform/dashboard/services/DashboardScenariosService.java [hunk 1]
```diff
@@ -48,6 +48,7 @@ public DashboardScenariosService(ScenarioRepository scenarioRepository,
     public void recordScenarios(String authorisation, String scenarioReference,
                                 String uniqueCaseIdentifier, ScenarioRequestParams scenarioRequestParams) {
 
+        Iterable<ScenarioEntity> scenarioBfyName = scenarioRepository.findAll();
```
Comment (neeta-hmcts) [inline dashboard-notifications/src/main/java/uk/gov/hmcts/reform/dashboard/services/DashboardScenariosService.java:51]: typo error

File: src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/ClaimantLiPResponse.java [hunk 2]
```diff
@@ -38,6 +38,11 @@ public boolean hasCourtDecisionInFavourOfClaimant() {
         return RepaymentDecisionType.IN_FAVOUR_OF_CLAIMANT == claimantCourtDecision;
     }
 
+    @JsonIgnore
+    public boolean hasCourtDecisionInFavourOfCDefendant() {
```
Comment (neeta-hmcts) [inline src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/ClaimantLiPResponse.java:42]: extra letter 'C' in function name

**Review by neeta-hmcts — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java [hunk 3]
```diff
@@ -29,6 +33,10 @@ public Map<String, Object> mapCaseDataToParams(CaseData caseData) {
             params.put("defendantAdmittedAmount", formatAmount(getDefendantAdmittedAmount(caseData)));
         }
 
+        if (nonNull(caseData.getRespondToClaimAdmitPartLRspec())) {
+            params.put("defendantAdmittedAmountPaymentDeadline", caseData.getRespondToClaimAdmitPartLRspec().getWhenWillThisAmountBePaid());
+        }
```
Comment (GorilaNaranja) [inline src/main/java/uk/gov/hmcts/reform/civil/service/DashboardNotificationsParamsMapper.java:38]: This value already exists. Check the next if. Also the value that exists is split in 2 to do the date translations for EN and CY as Dharmendra requested.

**Review by GorilaNaranja — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/utils/DateUtils.java [hunk 4]
```diff
@@ -14,6 +14,11 @@ private DateUtils() {
         //No op
     }
 
+    public static String formatDate(LocalDateTime date) {
```
Comment (GorilaNaranja) [inline src/main/java/uk/gov/hmcts/reform/civil/utils/DateUtils.java:17]: Duplicated method, exact the same as the next one

**Review by GorilaNaranja — CHANGES_REQUESTED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/utils/DateUtils.java (hunk 4 above)
Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/reform/civil/utils/DateUtils.java:17]: one is with LocalDate and the other is wit LocalDateTime

**Review by GorilaNaranja — APPROVED** [review]

**Review by dharmendrak — APPROVED** [review]

**Pull request merged**